db.go

```diff
@@ -40,6 +40,9 @@ const (
 // default page size for db is set to the OS page size.
 var defaultPageSize = os.Getpagesize()
 
+// The step used by the timeout mechanisms.
+const timeoutStep = 50 * time.Millisecond
```

Comment:
`const flockRetryTimeout = 50 * time.Millisecond`?

Reply:
Good for me. Will push an update